doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -654,9 +654,18 @@ EVP_PKEY_CTX_set_kem_op() sets the KEM operation to run. This can be set after
 EVP_PKEY_encapsulate_init() or EVP_PKEY_decapsulate_init() to select the kem
 operation. For the key types that support encapsulation and don't have the
 default operation, e.g. RSA, this function must be called before
-EVP_PKEY_encapsulate() or EVP_PKEY_decapsulate(). The supported values for the
-built-in algorithms are enumerated in L<EVP_KEM-RSA(7)>, L<EVP_KEM-EC(7)>,
-L<EVP_KEM-X25519(7)>, and L<EVP_KEM-X448(7)>.
+EVP_PKEY_encapsulate() or EVP_PKEY_decapsulate().
+
+=for comment Is this an "operation" or an "operating mode".  I would have
```

Comment:
This comment doesn't belong here in the docs - it is a separate issue ... I wouldn't merge docs with comments like this.